test/example_data.dart

```diff
@@ -495,6 +498,54 @@ UpdateMessageEvent updateMessageMoveEvent(
   );
 }
 
+/// [UpdateMessageEvent] where [origMessages] are moved to somewhere else.
+UpdateMessageEvent updateMessageEventMoveFrom({
+  required List<StreamMessage> origMessages,
+  String? newTopic,
+  int? newStreamId,
```

Comment:
Speaking of https://github.com/zulip/zulip-flutter/pull/787#discussion_r1696096924, do you think we should insert `propagateMode` and `flags` in between? Currently they are added to the end of the named parameters.

Reply:
I think at the end is fine — it seems like a logical place for those two. (`flags` is closely tied to `content`, which is at the end.)

Putting them in the middle, in the same order as on the event class's declaration, would also be fine. There's a bit of a tension because, ultimately, the order in the docs doesn't match a logical order; on the class we choose to match the docs.